tools/cold-backup/run.sh

```diff
@@ -331,32 +327,37 @@ restore() {
   if [[ ! -f $SOURCE_FILE ]]; then
     error "$SOURCE_FILE file not found"
   fi
-
-  # init context
-  local tmp="$BACKUP_PATH/$NAMESPACE/$CLUSTER_NAME"
+  
   check_kube_conn
+  if [[ -z $(kubectl get ns "$NAMESPACE" -o name --ignore-not-found) ]]; then
+    error "Namespace not found"
+  fi
+
   if [[ $CONFIRM == true ]]; then
     echo "Restore of cluster $NAMESPACE/$CLUSTER_NAME"
     confirm
   fi
-  uncompress $SOURCE_FILE $tmp
-  source $tmp/env
+  
+  local tmp="$BACKUP_PATH/$NAMESPACE/$CLUSTER_NAME"
+  uncompress "$SOURCE_FILE" "$tmp"
+  # shellcheck disable=SC1091
```

Comment:
Maybe you can add a short comment why is it needed into the code (I assume it cannot follow the code since it does nto exist)

Reply:
https://github.com/koalaman/shellcheck/wiki/SC1091

Reply:
Well, that is what I assumed. But would be still good to have it in the code.

Reply:
ok